nexus/utils.ts

```diff
@@ -162,10 +154,7 @@ export const processEditorDocument = (document: NodeType[]) => {
 
 // Takes in an original Post or Comment and a currently logged in User and checks that
 // the currentUser has permission to update or delete that Post/Comment
-export const hasPostPermissions = (
-  original: Post | Comment,
-  currentUser: User,
-) => {
+export const hasPostPermissions = (original: Post | Comment | PostComment, currentUser: User) => {
```

Comment:
The only substantive change in this file is adding the `PostComment` union

Reply:
To avoid having to continue to expand this union like this, we could change its signature like:

```
type AuthoredObject = { authorId: int }
export const hasAuthorPermissions(original: AuthoredObject, currentUser: User) {
...
```

Reply:
Done! Great idea!